UI/window-basic-main-scene-collections.cpp

```diff
@@ -244,7 +244,6 @@ void OBSBasic::RefreshSceneCollections()
 	ui->actionRemoveSceneCollection->setEnabled(count > 1);
 
 	OBSBasic *main = reinterpret_cast<OBSBasic*>(App()->GetMainWindow());
-
```

Comment:
This whitespace change seems unrelated to the rest of the PR.  You should revert this.